tests/functional/test_newsletter.py

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from ..pages.newsletter import NewsletterPage
+
+
+@pytest.mark.nondestructive
+def test_default_values(base_url, selenium):
+    page = NewsletterPage(base_url, selenium).open()
+    assert '' == page.email
+    assert 'United States' == page.country
+    assert 'English' == page.language
+    assert page.html_format_selected
+    assert not page.text_format_selected
+    assert not page.privacy_policy_accepted
+
+
+@pytest.mark.nondestructive
```

Comment:
Good idea. I'm going to enhance pytest-selenium to accept a sensitive url regex in setup.cfg. At the moment it requires either either a command line argument or environment variable. Supporting setup.cfg means it can easily be configured for a project and stored in the repo.